examples/server/utils.hpp

```diff
@@ -642,9 +642,30 @@ static json oaicompat_completion_params_parse(
         throw std::runtime_error("Cannot use custom grammar constraints with tools.");
     }
 
+    /* Prefill assistant message support */
```

Comment:
```suggestion
    // if the assistant message appears at the end of list, we do not add end-of-turn token
    // for ex. this can be useful to modify the reasoning process in reasoning models
```